src/platform/utilities/sso/index.js

```diff
@@ -25,19 +26,25 @@ export async function ssoKeepAliveSession() {
     // ttl is null, we can't determine if the user has a session or not
     localStorage.removeItem('hasSessionSSO');
   }
-  return authn;
+  return { type, authn };
 }
 
 export async function checkAutoSession(application = null, to = null) {
-  const authn = await ssoKeepAliveSession();
+  const { type, authn } = await ssoKeepAliveSession();
   if (hasSession() && hasSessionSSO() === false) {
     // explicitly check to see if the SSOe session is false, as it could also
     // be null if we failed to get a response from the SSOe server, in which
     // case we don't want to logout the user because we don't know
     logout('v1', 'sso-automatic-logout');
-  } else if (!hasSession() && hasSessionSSO() && !getForceAuth() && authn) {
-    const params = { inbound: 'true', authn };
-    login('custom', 'v1', application, to, params, 'sso-automatic-login');
+  } else if (!hasSession() && hasSessionSSO() && !getForceAuth() && type) {
+    // only attempt an auto login if the user is
+    // a) does not have a VA.gov session
+    // b) has an SSOe session
+    // c) is not required for forceAuth (meaning their environment has SSOe
+    //    enabled and they have not previously tried to login)
```

Comment:
this is tangential to these updates, but I think we still need to potentially address `setForceAuth` getting prematurely called to match the "have not previously tried to login" aspect of this

Reply:
@hinzed1127 are you talking about this block?  https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/user-nav/containers/AutoSSO.jsx#L30-L34

Reply:
yup that's the one. since right now that would require someone to log in normally once, after `inboundSSOe` is enabled, before they actually would have an inbound session triggered